eng/common/cross/toolchain.cmake

```diff
@@ -37,6 +37,13 @@ elseif(TARGET_ARCH_NAME STREQUAL "arm")
   if(TIZEN)
     set(TIZEN_TOOLCHAIN "armv7hl-tizen-linux-gnueabihf/9.2.0")
   endif()
+elseif(TARGET_ARCH_NAME STREQUAL "armv6")
```

Comment:
This file in arcade repo does not have this change, so your changes in `eng/common` (mirror directory) will be overwritten by the next arcade update.

Reply:
https://github.com/dotnet/arcade/pull/8326